test/rvm_message_bus.test.ts

```diff
@@ -0,0 +1,110 @@
+/*
+Copyright 2017 OpenFin Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import * as assert from 'assert';
+import * as mockery from 'mockery';
+import {EventEmitter} from 'events';
+const App = require('electron').app;
+
+App.generateGUID = Math.random;
+
+const mockWMCopyData  = {
+    WMCopyData: function(){
+        return {
+            on: (x: any) => x,
+            publish: (x: any ) => x
```

Comment:
There quite a bit of additional linting here... https://appoji.jira.com/browse/RUN-3035